_posts/en/newsletters/2019-10-23-newsletter.md

```diff
@@ -0,0 +1,130 @@
+  people through review of the proposed [bip-schnorr][],
+  [bip-taproot][], and [bip-tapscript][] changes.  All developers,
+  academics, and anyone else with technical experience are welcome.  The
+  minimum expected commitment is four hours a week for seven weeks, with
+  one hour each week being a group meeting and the other three hours
+  being your own independent review of the proposals.  In addition to
+  review, developers will be encouraged to optionally implement a
+  proof-of-concept that either roughly integrates schnorr or taproot
+  into existing software or which demonstrates the new or improved
+  features the proposals make possible in Bitcoin and Lightning,
+  allowing implementers to identify flaws or sub-optimal requirements in
+  the current proposals that might be missed by people who only read the
+  documentation.
+
+    The ultimate goal of the review club is to allow participants to
+    gain enough technical familiarity with the proposals to be able to
+    either vocally support the proposals, advocate for changes to the
+    proposals, or lucidly explain why the proposals shouldn't be adopted
+    into the Bitcoin consensus rules.  Changing Bitcoin is something
+    that should be done carefully, and it's in every user's interest
+    that a large number of technical reviewers examine the proposals for
+    possible flaws before they are implemented and before users are
+    asked to consider upgrading their full nodes to enforce the new
+    rules---a change that can't be safely undone for as long as the
+    rules affect anyone's bitcoins.  Whether through the review club or
+    in some other way, Optech strongly encourages all technically
+    skilled Bitcoin users to dedicate time to reviewing the taproot set
+    of proposals.
+
+    Anyone wanting to participate should RSVP soon so that the
+    organizers can estimate the total number of participants and start
+    forming study groups.  To register or learn more, please see
+    [FIXME][].
+
+## Changes to services and client software
+
+*In this monthly feature, we highlight interesting updates to Bitcoin wallets
+and services.*
+
+- **Electrum Lightning support:** In a [series of commits this
+  month][electrum commits], Electrum has merged support for Lightning
+  Network into master. A presentation deck titled [Lightning Implementation in
+  Electrum][electrum lightning presentation] by Thomas Voegtlin provides some
+  background information and screenshots.
+
+- **Blockstream Green Tor support:** Version 3.2.4 of the Blockstream Green
+  Wallet [adds built-in Tor support][blockstream green tor article] for both iOS and Android.
+  While Tor was supported in previous Android versions, it required a separate
+  application whereas both Android and iOS versions are now bundled with Tor
+  support.
+
+## Notable code and documentation changes
+
+*Notable changes this week in [Bitcoin Core][bitcoin core repo],
```

Comment:
I've never noticed this before. We list LND before C-Lightning and Eclair. I'd have a slight preference to change that to be sorted alphabetically to signal that we don't have any preference between lightning implementations.

Reply:
> I've never noticed this before. We list LND before C-Lightning and Eclair. I'd have a slight preference to change that to be sorted alphabetically to signal that we don't have any preference between lightning implementations.

Excellent suggestion.